apps/oc_chef_wm/src/chef_wm_sandboxes.erl

```diff
@@ -71,8 +71,9 @@ auth_info('POST', Req, State) ->
 
 create_path(Req, #base_state{organization_guid = OrgId,
                              resource_state = SandboxState}=State) ->
-    %% there is no name, to help with uniqueness, maybe take first checksum or digest of content?
-    Name_ish = crypto:md5(wrq:req_body(Req)),
+    %% there is no name so to help with uniqueness, we take a digest of
+    %% the content
+    Name_ish = crypto:hash(md5, wrq:req_body(Req)),
```

Comment:
Thank you! That warning has been bugging me forever, but I keep forgetting to update :D